monitoring/ingestion/populator-dashboard.json

```diff
@@ -0,0 +1,660 @@
+{
+  "__inputs": [
+    {
+      "name": "DS_PROMETHEUS_SCALITY",
+      "label": "prometheus_scality",
+      "description": "",
+      "type": "datasource",
+      "pluginId": "prometheus",
+      "pluginName": "Prometheus"
+    },
+    {
+      "name": "VAR_JOB",
+      "type": "constant",
+      "label": "job",
+      "value": "populator",
```

Comment:
Would you mind changing the job's value for: `artesca-data-backbeat-ingestion-producer-headless`.
It seems to be hardcoded.